tests/ParserFrameworkValidationTests.php

```diff
@@ -18,9 +18,11 @@ public function frameworkErrorProvider() {
             $iterator = new RecursiveDirectoryIterator(__DIR__ . "/../validation/frameworks/" . $frameworkName);
 
             foreach (new RecursiveIteratorIterator($iterator) as $file) {
-                if (strpos($file, ".php") !== false) {
+                $pathName = $file->getPathname();
+                if (preg_match('@\.php$@', $pathName) > 0) {
```

Comment:
Could you use `/` as the regex delimiter instead?

Reply:
Done